.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+1. Describe what you expected to happen.
+
+2. Describe what actually happened.
+
+3. Describe what version of Pulp and what OS is affected by the
+   issue.
```

Comment:
Nitpick: I don't think this text needs to be line-wrapped.

Reply:
@Ichimonji10, thanks for your review. All comments were addressed.